cranelift/frontend/src/frontend.rs

```diff
@@ -223,7 +223,7 @@ pub enum DefVariableError {
     ///
     /// note: to obtain the type of the value, you can call
     /// [`cranelift_codegen::ir::dfg::DataFlowGraph::value_type`] (using the
-    /// [`FunctionBuilder.func.dfg`] field)
+    /// `FunctionBuilder.func.stencil.dfg` field)
```

Comment:
`Function` derefs to `FunctionStencil`, so the extra `.stencil` is not necessary.

Reply:
Alright, I'll remove it!